### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by ali-rostami**
If you search the "style of the image" through the library, there are some other places that have this expression and need an update too. (e.g `lv_canvas_draw_img`)

### CODE REVIEW

**Comment by kisvegabor** [comment]
Thank you!

The comments of [canvas](https://github.com/lvgl/lvgl/blob/master/src/lv_widgets/lv_canvas.c#L847) were recently fixed. Or did you mean someting else?

File: src/lv_draw/lv_img_cache.c [hunk 1]
```diff
@@ -59,7 +59,7 @@ static uint16_t entry_cnt;
  * The image will be left open meaning if the image decoder open callback allocated memory then it will remain.
  * The image is closed if a new image is opened and the new image takes its place in the cache.
  * @param src source of the image. Path to file or pointer to an `lv_img_dsc_t` variable
- * @param style style of the image
+ * @param color The color of the image
```
Comment (kisvegabor) [inline src/lv_draw/lv_img_cache.c]:
```suggestion
 * @param color The color of the image with `LV_IMG_CF_ALPHA_...`
```

File: src/lv_draw/lv_img_cache.h [hunk 2]
```diff
@@ -46,7 +46,7 @@ typedef struct {
  * The image will be left open meaning if the image decoder open callback allocated memory then it will remain.
  * The image is closed if a new image is opened and the new image takes its place in the cache.
  * @param src source of the image. Path to file or pointer to an `lv_img_dsc_t` variable
- * @param style style of the image
+ * @param color The color of the image
```
Comment (kisvegabor) [inline src/lv_draw/lv_img_cache.h]:
```suggestion
 * @param color The color of the image with `LV_IMG_CF_ALPHA_...`
```

File: src/lv_draw/lv_img_decoder.c [hunk 3]
```diff
@@ -114,7 +114,7 @@ lv_res_t lv_img_decoder_get_info(const char * src, lv_img_header_t * header)
  *  1) File name: E.g. "S:folder/img1.png" (The drivers needs to registered via `lv_fs_add_drv()`)
  *  2) Variable: Pointer to an `lv_img_dsc_t` variable
  *  3) Symbol: E.g. `LV_SYMBOL_OK`
- * @param style the style of the image
+ * @param color the color of the image
```
Comment (kisvegabor) [inline src/lv_draw/lv_img_decoder.c]:
```suggestion
 * @param color The color of the image with `LV_IMG_CF_ALPHA_...`
```

File: src/lv_draw/lv_img_decoder.h [hunk 4]
```diff
@@ -157,7 +157,7 @@ lv_res_t lv_img_decoder_get_info(const char * src, lv_img_header_t * header);
  *  1) File name: E.g. "S:folder/img1.png" (The drivers needs to registered via `lv_fs_add_drv()`)
  *  2) Variable: Pointer to an `lv_img_dsc_t` variable
  *  3) Symbol: E.g. `LV_SYMBOL_OK`
- * @param style the style of the image
+ * @param color the color of the image
```
Comment (kisvegabor) [inline src/lv_draw/lv_img_decoder.h]:
```suggestion
 * @param color The color of the image with `LV_IMG_CF_ALPHA_...`
```

**Comment by ali-rostami** [comment]
> Thank you!
> 
> The comments of [canvas](https://github.com/lvgl/lvgl/blob/master/src/lv_widgets/lv_canvas.c#L847) were recently fixed. Or did you mean someting else?

Oh, I didn't see the new changes.
But take look to [this](https://github.com/lvgl/lvgl/blob/2068ab3211ab4df04f283f39c7b6a4b58c64ee20/src/lv_draw/lv_img_cache.c#L65), [this](https://github.com/lvgl/lvgl/blob/2068ab3211ab4df04f283f39c7b6a4b58c64ee20/src/lv_draw/lv_img_cache.h#L52), [this](https://github.com/lvgl/lvgl/blob/2068ab3211ab4df04f283f39c7b6a4b58c64ee20/src/lv_draw/lv_img_decoder.c#L121) and [this](https://github.com/lvgl/lvgl/blob/2068ab3211ab4df04f283f39c7b6a4b58c64ee20/src/lv_draw/lv_img_decoder.h#L164).

**Comment by kisvegabor** [comment]
> But take look to this, this, this and this.

I see, but these are fixed in your PR, right?

**Comment by ali-rostami** [comment]
Oh, yeah, my mistake. sorry 🤔
I'm not good at using Github.

**Comment by kisvegabor** [comment]
No problem, feel free to ask. I happily help if I can.

We can merge this PR if you reopen it.

**Comment by ali-rostami** [comment]
A question that I have is that what would happen If I delete the **patch-1** in my repository after you merge it?

**Comment by kisvegabor** [comment]
You can delete it. It's already copied to LVGL.

**Pull request merged**